docs/development/workflow/development_workflow.rst

```diff
@@ -168,6 +168,13 @@ our coding style standards. This can be done by installing ``pre-commit`` in the
 of your astropy repository by running::
 
     pip install pre-commit
+
+Or if you prefer `conda`_::
+
+    conda install pre-commit
```

Comment:
@WilliamJamieson or @nstarman , would you recommend pre-commit from conda or should it always be pip?

Reply:
I don't use conda so I do not know how well this works.

Reply:
pre-commit from conda works great (provided of course that one is working in a conda environment).

Reply:
If one is using conda, all things should be installed via conda if they can. pip + conda doesn't always play nice.